icu4c/source/common/lstmbe.cpp

```diff
@@ -639,6 +639,7 @@ LSTMBreakEngine::divideUpDictionaryRange( UText *text,
                                                 int32_t startPos,
                                                 int32_t endPos,
                                                 UVector32 &foundBreaks,
+                                                UBool isPhraseBreaking,
```

Comment:
If you got compiler error, change to 
                             UBool /* isPhraseBreaking */,